programs/biome.nix

```diff
@@ -335,9 +366,8 @@ in
             filterEmpty (filterDefaults cfg.settings);
         in
         l.optionals (settings != { }) [
-          # NOTE(@huwaireb): Biome does not accept a direct path to a file for config-path, only a directory.
           "--config-path"
-          (l.toString (p.writeTextDir "biome.json" (l.toJSON settings)))
```

Comment:
This got resolved in newer version of biome?

Reply:
Yep, since [v1.7.0 (2024-04-15)](https://biomejs.dev/internals/changelog/#new-features-19)

> Support passing a file path to the `--config-path` flag or the `BIOME_CONFIG_PATH` environment variable.